src/Infrastructure/Controller/Regulation/RegulationDeleteController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Controller\Regulation;
+
+use App\Application\CommandBusInterface;
+use App\Application\Regulation\Command\DeleteRegulationCommand;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+final class RegulationDeleteController
+{
+    public function __construct(
+        private \Twig\Environment $twig,
+        private CommandBusInterface $commandBus,
+    ) {
+    }
+
+    #[Route(
+        '/regulations/{uuid}',
+        name: 'app_regulation_delete',
+        requirements: ['uuid' => '[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}'],
+        methods: ['DELETE'],
+    )]
+    public function __invoke(string $uuid): Response
```

Comment:
Il faudrait rajouter ici une protection CSRF token. Par exemple https://symfony.com/doc/current/security/csrf.html#generating-and-checking-csrf-tokens-manually

Reply:
On n'a pas de protection CSRF sur tous les formulaires partout par défaut ?